examples/sandbox/index.html

```diff
@@ -89,7 +89,7 @@ <h4>Select</h4>
     <h4>DebugLink</h4>
     <button class='button-outline' id='yes'>Yes</button>
     <button class='button-outline' id='no'>No</button>
-    <button class='button-outline' id='cancel'>Cancel</button>
+    <button class='button-outline cancel'>Cancel</button>
```

Comment:
hmmmm?

Reply:
@keepkeyjon I just changed this for ease of use. Definitely not married to it at all

Reply:
was confused by it at first, but now it makes sense... carry on.